localization/keys/en-us/5e-SRD-Weapon-Properties.json

```diff
@@ -0,0 +1,25 @@
+{
+  "ammunitionName": "Ammunition",
+  "ammunitionDesc1": "You can use a weapon that has the ammunition property to make a ranged attack only if you have ammunition to fire from the weapon. Each time you attack with the weapon, you expend one piece of ammunition. Drawing the ammunition from a quiver, case, or other container is part of the attack (you need a free hand to load a one-handed weapon).",
+  "ammunitionDesc2": "At the end of the battle, you can recover half your expended ammunition by taking a minute to search the battlefield. If you use a weapon that has the ammunition property to make a melee attack, you treat the weapon as an improvised weapon (see \"Improvised Weapons\" later in the section). A sling must be loaded to deal any damage when used in this way." ,
+  "finesseName": "Finesse",
+  "finesseDesc1": "When making an attack with a finesse weapon, you use your choice of your Strength or Dexterity modifier for the attack and damage rolls. You must use the same modifier for both rolls.",
+  "heavyName": "Heavy",
+  "heavyDesc1": "Small creatures have disadvantage on attack rolls with heavy weapons. A heavy weapon's size and bulk make it too large for a Small creature to use effectively.",
+  "lighName": "Light",
```

Comment:
Should this be `lightName` instead of `lighName`?

Reply:
Yes. Freaking typo haha